packages/ibm-products-styles/src/components/TagSet/_tag-set.scss

```diff
@@ -13,41 +13,43 @@
 @use '../../global/styles/project-settings' as *;
 @use '../../global/styles/mixins' as *;
 
-$_block-class: #{$pkg-prefix}--tag-set;
-$block-class-overflow: #{$_block-class}-overflow;
-$block-class-modal: #{$_block-class}-modal;
+$block-class: #{$pkg-prefix}--tag-set;
+$block-class-overflow: #{$block-class}-overflow;
+$block-class-modal: #{$block-class}-modal;
 
-@include block-wrap($_block-class) {
-  &.#{$_block-class} {
+@include block-wrap($block-class) {
+  &.#{$block-class} {
     display: block;
     inline-size: 100%;
   }
 
-  .#{$_block-class}__space {
+  .#{$block-class}__space {
     position: relative;
     display: block;
     inline-size: 100%;
     white-space: nowrap;
   }
 
-  .#{$_block-class}__space--align-end {
+  .#{$block-class}__space--align-end {
     text-align: end;
   }
 
-  .#{$_block-class}__space--align-center {
+  .#{$block-class}__space--align-center {
     text-align: center;
   }
 
-  .#{$_block-class}__tag-container {
+  .#{$block-class}__tag-container {
     display: inline-flex; // needed to register a width
+    // Spacing between tags.
+    gap: $spacing-02;
```

Comment:
Don't include the `gap` spacing if you're going to add the margin spacing back to the tags. Otherwise, there ends up being 12px of gap between each tag instead of 8px.